src/azure-cli-core/azure/cli/core/aaz/_arg_fmt.py

```diff
@@ -766,3 +767,30 @@ def read_file(self, file_path):
             except UnicodeDecodeError:
                 data = str(base64_data)
         return data
+
+
+class AAZPaginationTokenArgFormat(AAZBaseArgFormat):
+    def __call__(self, ctx, value):
+        def validate_json(s):
+            try:
+                obj = json.loads(s)
+                assert isinstance(obj, dict)
```

Comment:
Please not use assert here, because the obj is passed in by customer, you should raise AAZInvalidArgValueError when it's not a dict

Reply:
Maybe we should also verify the properties of obj here